src/Altinn.Notifications.Email.Core/Status/EmailSendResultMapper.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Altinn.Notifications.Email.Core.Status;
+
+/// <summary>
+/// Mapper handling parsing to EmailSendResult
+/// </summary>
+public static class EmailSendResultMapper
+{
+    /// <summary>
+    /// Parse AcsEmailDeliveryReportStatus to EmailSendResult
+    /// </summary>
+    /// <param name="deliveryStatus">Delivery status from Azure Communication Service</param>
+    /// <returns></returns>
+    /// <exception cref="ArgumentException">Throws exception if unknown delivery status</exception>
+    public static EmailSendResult ParseDeliveryStatus(string? deliveryStatus)
```

Comment:
I guess this mapper could be moved to the API project as it is the only place the mapper is used.

Reply:
Moved